public/app/features/explore/PromQueryField.tsx

```diff
@@ -421,7 +421,7 @@ class PromQueryField extends React.PureComponent<PromQueryFieldProps, PromQueryF
     const containsMetric = selector.indexOf('__name__=') > -1;
     const existingKeys = parsedSelector ? parsedSelector.labelKeys : [];
 
-    if ((text && text.startsWith('=')) || _.includes(wrapperClasses, 'attr-value')) {
+    if ((text && text.match(/^[!=]/)) || _.includes(wrapperClasses, 'attr-value')) {
```

Comment:
Can we use a similar operator regex as above? Eg. `^!?=~?` ?

Reply:
Yep, absolutely—I was being frugal with the match here, seeing as though the label matching operators can only ever start with `!` or `=` (at present anyway).